.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
```

Comment:
@awoods I would add that I have no particular commitment to using `html5validator`, but it catches a lot of simple HTML errors and seems to work well. If there is another tool that is better, that's fine, but `html5validator` certainly does the job.